main.py

```diff
@@ -95,13 +98,13 @@
 
 # Read the testing dataset from a TSV file, specifying the column separator as tab
 test_df = pd.read_csv(
-    r"datasets\drugsComTest_raw.tsv", sep="\t", quoting=2, dtype=dtypes
+    r"datasets\drugsComTest_raw.tsv", quoting=2, dtype=dtypes, on_bad_lines = "skip"
 )
 
 # Convert the "date" column to datetime format in the training and testing datasets
 train_df["date"], test_df["date"] = pd.to_datetime(
-    train_df["date"], format="%B %d, %Y"
-), pd.to_datetime(test_df["date"], format="%B %d, %Y")
+    train_df["date"], format="%d-%b-%y"
+), pd.to_datetime(test_df["date"], format="%d-%b-%y")
```

Comment:
Any reason for changing the format? @Soumya1219

Reply:
Yes because while I was trying to run the code it shows an error. So I changed the format but it is not related to my raised issue but as it showed me an error so I made some changes.

Reply:
Will this format change not cause error for other models in the code? Have you checked this @Soumya1219

Reply:
Yes as far as I knew I checked with other models so far it didn't effect other models. If there is such other conflict I will be change into original code. It's fine.

Reply:
Okay, you can do run the full code or other 2-3 models and check that its running flawlessly @Soumya1219